kinbot/mess.py

```diff
@@ -683,7 +683,7 @@ def write_submitscript(self, submitscript, uq_iter):
         with open(q_file) as f:
             tpl_head = f.read()
 
-        q_file = pkg_resources.resource_filename('tpl', self.par['queuing'] + '_mess_uq.tpl')
+        q_file = pkg_resources.resource_filename('tpl', self.par['queuing'] + '_mess.tpl')
```

Comment:
is this intentional?

Reply:
This line will be fixed. I was doing testing with KinBot on Menten, where this code was written. This file as well as a couple of others had a few lines I forgot to edit out when I made the commit for the work with Cl.

Reply:
ok, I accept this for now, but if it needs to be fixed, pls do it